dist/css-layout.h

```diff
@@ -724,30 +725,41 @@ static void layoutNodeImpl(css_node_t *node, float parentMaxWidth, css_direction
 
   if (isMeasureDefined(node)) {
     float width = CSS_UNDEFINED;
+    float height = CSS_UNDEFINED;
+    float parentMaxHeight = parentMaxWidth;
```

Comment:
The idea is to use this for images also, so that one could create a custom component with support for fixed aspect ratio. So given a width, the height would be calculated to maintain the aspect ratio of the image.

Currently this works only to calculate the height, given a width. I'm trying to make it work the other way also, that given a height, it should calculate width.

Think of it more like `- (CGSize *)sizeThatFits:(CGSize)size` than `measure`. Meaning it allows custom components to override layout.